bin/remove.sh

```diff
@@ -1,26 +1,32 @@
-#!/bin/sh -f
+#!/bin/bash -f
 #
 # Used to remove indexes for a particular terminology/version.
 #
 help=0
 config=1
 ncflag=""
-stardog=0
+graph_db=0
 es=0
+
+l_graph_db_type=${GRAPH_DB_TYPE:-"stardog"}
+l_graph_db_home=""
+l_graph_db_username=""
+l_graph_db_password=""
+
 while [[ "$#" -gt 0 ]]; do case $1 in
     --help) help=1;;
     # use environment variable config (dev env)
     --noconfig) config=0; ncflag="--noconfig";;
-    # remove stardog data
-    --stardog) stardog=1;;
+    # remove graph_db data
+    --graph_db) graph_db=1;;
     # remove es data
```

Comment:
Same comment as for list.sh.  Let's call it "graphdb" so it's easier to use without having to type an underscore